admin/class-database-proxy.php

```diff
@@ -16,23 +16,38 @@ class WPSEO_Database_Proxy {
 	/** @var bool */
 	protected $suppress_errors = true;
 
+	/** @var bool */
+	protected $is_global = false;
+
 	/** @var bool */
 	protected $last_suppressed_state;
 
 	/** @var wpdb */
 	protected $database;
 
 	/**
-	 * Sets the class attributes.
+	 * Sets the class attributes and registers the table.
 	 *
 	 * @param wpdb   $database        The database object.
 	 * @param string $table_name      The table name that is represented.
 	 * @param bool   $suppress_errors Should the errors be suppressed.
+	 * @param bool   $is_global       Should the table be global in multisite.
```

Comment:
`is_global` has a different meaning to me that the intended use.
Would prefer to have something that references `multisite` in the name.